cypress/integration/page.spec.js

```diff
@@ -1,7 +1,132 @@
-// WRITE TESTS HERE
+/// <reference types="cypress" />
 
 describe('Page', () => {
-  it('should be visitable', () => {
-    cy.visit('/');
+  const apiUrl = 'https://mate.academy/students-api/'
+
+  beforeEach(() => {
+    cy.visit('/')
+  });
+
+  it('user details should be updated after selecting a different user', () => {
+    cy.intercept(`${apiUrl}` + 'users/5', { fixture: 'user5' });
+    cy.intercept(`${apiUrl}` + 'users/6', { fixture: 'user6' });
+
+    cy.get('button')
```

Comment:
Selector is still too general

Reply:
You can add `data-cy` attribute to **existing** HTML in TodoList(to the root element or another) and just ask student to keep it.
And you will have a way stronger selector

Reply:
added data-cy